File: Feitos em C++ Parte 2/2061.cpp
#include<iostream>
#include<stdio.h>
using namespace std;
int main(){
    int a,b;
    cin>>a>>b;
    string name[b];
    for(int x=0;x<b;x++){
        cin>>name[x];
        if (name[x]=="fechou"){
            a++;
        }
        else{
            a--;
        }
    }
    cout<<a<<endl;
    return 0;
}

File: Feitos em Java/2061.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        int a=entrada.nextInt(),b=entrada.nextInt();
        String name;
        for(int x=0;x<b;x++){
            name=entrada.nextLine();
            if (name.equals("fechou")){
                a++;
            }
            else{
                a--;
            }
        }
        System.out.println(a); 
    }
}